CorsixTH/Lua/cheats.lua

```diff
@@ -169,3 +169,50 @@ function Cheats:cheatDecreasePrices()
     end
   end
 end
+
+--! Cheat fax function
+function Cheats:cheatByFax()
+  local cheatWindow = self.hospital.world.ui:getWindow(UICheats)
+  self:announceCheat()
+  -- If a cheats window is open, make sure the UI is updated
+  if cheatWindow then
+    cheatWindow:updateCheatedStatus()
+  end
+end
+
+--[[Cheats operated through Faxes go here]]
+--! Cheat that disables/enables Roujin's challenge (spawn rate cheat)
+function Cheats:cheatRoujin()
+  -- Roujin's challenge cheat
+  local hosp = self.hopsital
+  local ui = hosp.world.ui
+  if not hosp.spawn_rate_cheat then
```

Comment:
Instead of throwing lots of random names in the hospital name space, add a table with a useful name (eg `done_cheats`), and store this information in there.

Reply:
or `active_cheats` ?

Reply:
Still inside hospital I'm guessing? So `hospital.active_cheats` ?